File: ekyc/src/main/java/com/lib/ekyc/presentation/utils/face/interfaces/FrameReturn.java
package com.lib.ekyc.presentation.utils.face.interfaces;

import android.graphics.Bitmap;

import com.google.firebase.ml.vision.face.FirebaseVisionFace;
import com.lib.ekyc.presentation.utils.face.common.FrameMetadata;
import com.lib.ekyc.presentation.utils.face.common.GraphicOverlay;


public interface FrameReturn{
    void onFrame(
            Bitmap image ,
            FirebaseVisionFace face ,
            FrameMetadata frameMetadata,
            GraphicOverlay graphicOverlay
    );
}

File: ekyc/src/main/java/com/lib/ekyc/presentation/utils/face/interfaces/FrameReturn.kt
package com.lib.ekyc.presentation.utils.face.interfaces

import android.graphics.Bitmap
import com.google.firebase.ml.vision.face.FirebaseVisionFace
import com.lib.ekyc.presentation.utils.face.common.FrameMetaDataKT
import com.lib.ekyc.presentation.utils.face.common.GraphicOverlay


interface FrameReturn {
    fun onFrame(
        image: Bitmap?,
        face: FirebaseVisionFace?,
        frameMetadata: FrameMetaDataKT?,
        graphicOverlay: GraphicOverlay?
    )
}